projects/admin/src/app/service/extra.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
```

Comment:
perhaps extra.service is not the best choice name. Something like, editor service or similar is better.